Makefile

```diff
@@ -47,6 +47,10 @@ test-compile: check-links
 test-compile-changed: os-autoinst/
 	export PERL5LIB=${PERL5LIB_} ; for f in `git diff --name-only | grep '.pm'` ; do perl -c $$f 2>&1 | grep -v " OK$$" && exit 2; done ; true
 
+.PHONY: test-yamllint
+test-yamllint:
+	yamllint --strict $$(git ls-files "*.y*ml")
```

Comment:
We'll end up with yaml, yml, yxml... you get the idea :), better make it explicit and stick to `yaml`?